@bkmgit [wrote](#commitcomment-49837890):

> ```markdown
> [coc]: https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html
> [coc-reporting]: https://docs.carpentries.org/topic_folders/policies/incident-reporting.html
> ```
> are in `links.md` - why do they need to be included again in `CODE_OF_CONDUCT.md`?

I agree that this is an unfortunate repetition. Without making these definitions, `make lesson-check` throws an error:
```bash
./CODE_OF_CONDUCT.md: Internally-defined links may be missing definitions: "Carpentry Code of Conduct"=>"coc", "reporting guidelines"=>"coc-reporting"
make: *** [Makefile:134: lesson-check] Error 1
```

--- a/_config.yml
+++ b/_config.yml
@@ -87,6 +87,7 @@ exclude:
   - bin/
   - .Rproj.user/
   - .vendor/
+  - vendor/
   - .docker-vendor/
 
 # Turn on built-in syntax highlighting.

